sql/api/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -4263,6 +4263,14 @@ object functions {
   def sentences(string: Column, language: Column, country: Column): Column =
     Column.fn("sentences", string, language, country)
 
+  /**
+   * Splits a string into arrays of sentences, where each sentence is an array of words.
```

Comment:
We should probably mention what's the default value for `country`

Reply:
Updated.

When we construct Locale in  `new Locale(`language`)` format, its `country` value is `""`
<img width="464" alt="image" src="https://github.com/user-attachments/assets/ea315292-dfb6-4ec6-9316-44f497e5f272">

Reply:
<img width="361" alt="image" src="https://github.com/user-attachments/assets/538d9eee-0f6d-41e0-af7a-6158136c408e">